recipe/meta.yaml

```
@@ -1,61 +1,79 @@
-{% set dependency_fastavro = "fastavro >=0.21.2" %}
-{% set dependency_pandas = "pandas >=0.17.1" %}
-{% set dependency_pyarrow = "pyarrow >=0.15.0" %}
 {% set name = "google-cloud-bigquery-storage" %}
-{% set version = "2.1.0" %}
-{% set sha256 = "7da8691d218e494e2688f3c72cf4249d504b0a55fe41cb1ac6e7903c1e03a3d4" %}
+{% set version = "2.26.0" %}
+{% set sha256 = "840275bd0a4b207c0ac821bcc6741406ffb3b5ef940fa05089a90eb4403453fa" %}
 
 package:
   name: {{ name|lower }}-split
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
   number: 0
-  noarch: python
+  skip: true  # [py<38]
 
 requirements:
   host:
     - python
     - pip >=18.1
     - setuptools >=34.0.0
+    - wheel
+  run:
+    - python
 
 outputs:
   - name: {{ name }}
     build:
-      noarch: python
+      skip: true  # [py<38]
```

```suggestion
      skip: true  # [py<37]
```

`python_requires=">=3.7"`??

---

we could, but in this case I followed what conda-forge is doing. The whole group of `google-cloud-*` is set to py<38 read also comment about `importlib-metadata` (also avoided in other dependencies)

```suggestion
      # we skip py<38 following along conda-forge, to avoid importlib-metadata
      # which is needed for py37
      skip: true  # [py<38]
```

---

Yep, all good.  As I noted on the other one, if you differ from upstream, people are going to ask questions unless you say why.